<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewCtrls</class>
 <widget class="QWidget" name="ReviewCtrls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="RangeSlider" name="reviewSlider">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>59</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Slider adjusts the playback position</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Starts playback from the current position</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Media/media.qrc">
     <normaloff>:/Review_Ctrl_Play_96.png</normaloff>:/Review_Ctrl_Play_96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="repeat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>55</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Auto restarts the log playback upon completion</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Media/media.qrc">
     <normaloff>:/Review_Ctrl_Repeat_96.png</normaloff>:/Review_Ctrl_Repeat_96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="speed">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Adjusts the speed of playback</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Media/media.qrc">
     <normaloff>:/Review_Ctrl_Speed_96.png</normaloff>:/Review_Ctrl_Speed_96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>17</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="playbackSpeed">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>15</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
  </widget>
  <widget class="QLabel" name="playbackTime">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>68</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exportVid">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>55</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Export the section of log file to a video</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Media/media.qrc">
     <normaloff>:/Review_Func_Export_96.png</normaloff>:/Review_Func_Export_96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RangeSlider</class>
   <extends>QSlider</extends>
   <header>..\Controls\RangeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Media/media.qrc"/>
 </resources>
 <connections/>
</ui>
